Source/Dafny/Verifier/Translator.Statement.cs

```diff
@@ -0,0 +1,2146 @@
+using System;
```

Comment:
Ditto on this file name, I'd at least call it something that implies it's a collection of methods rather than defining `Statement` itself.